run-tests.pl

```diff
@@ -38,9 +38,14 @@
    search_path => [ "SyTest::Output" ],
    require     => 1;
 
+use Module::Pluggable
+   sub_name    => "homeserver_factories",
+   search_path => [ "SyTest::HomeserverFactory" ],
```

Comment:
I'd change this to

    sub_name  => "homeserver_classes",
    search_path => [ "SyTest::Homeserver" ],